samples/IntegrateWithOtherPushProvider/src/main/java/com/swrve/sdk/demo/ConfigContentFragment.java

```diff
@@ -0,0 +1,51 @@
+package com.swrve.sdk.demo;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import com.swrve.sdk.SwrveSDK;
+
+import java.net.URISyntaxException;
+
+public class ConfigContentFragment extends Fragment {
```

Comment:
I would remove this class completely and just have a MainActivity (same for the AdvancedPushNotifications project). There should be no UI in the app. The MainActivity should have text to describe what this sample is doing.